ccmlib/scylla_repository.py

```diff
@@ -7,11 +7,13 @@
 import subprocess
 import re
 import gzip
+import sys
 
 from subprocess import Popen, PIPE, STDOUT
 
 import hashlib
 import requests
+from pathlib import Path
```

Comment:
this breaks python2 compatibility, there are still a few places which are running with python2.
can you switch to `os.listdir` or add the backport `pathlib2` to the the `setup.py`

Reply:
see:
https://www.python.org/dev/peps/pep-0508/
for how to add it to setup.py for version < 3.2

Reply:
Ah sorry, didn't realize we need python2 compatibility here.
I can drop pathlib then.

Reply:
Where do we use python2? Can we drop it? it's dead. An ex-python.

Reply:
Pushed python2 compatible version.

Reply:
@avikivity it's dead, but I thought we have some zombies left....
I've went over all of them, and seem like I've moved all the places to use pyhon3.7 already....

so it was just a habit I guess...

I would want to remove the python 2.7 classifiers our of `setup.py` to announce this doesn't support python2 anymore